pkg/collector/py/datadog_agent.go

```diff
@@ -151,6 +151,8 @@ func LogMessage(message *C.char, logLevel C.int) *C.PyObject {
 		log.Info(goMsg)
 	case 10: // DEBUG
 		log.Debug(goMsg)
+	case 7: // TRACE
+		log.Trace(goMsg)
```

Comment:
Also, I think it's worth mentioning in an inline comment that this is a custom log level, and say where it's defined